src/main/java/ch/njol/skript/expressions/ExprOnlinePlayersCount.java

```diff
@@ -18,42 +18,44 @@
  */
 package ch.njol.skript.expressions;
 
-import org.bukkit.Bukkit;
-import org.bukkit.event.Event;
-import org.bukkit.event.server.ServerListPingEvent;
-import org.eclipse.jdt.annotation.Nullable;
-
-import com.destroystokyo.paper.event.server.PaperServerListPingEvent;
 import ch.njol.skript.Skript;
-import ch.njol.skript.bukkitutil.PlayerUtils;
 import ch.njol.skript.classes.Changer.ChangeMode;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.lang.util.SimpleExpression;
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
+import com.destroystokyo.paper.event.server.PaperServerListPingEvent;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.bukkit.event.server.ServerListPingEvent;
+import org.eclipse.jdt.annotation.Nullable;
 
 @Name("Online Player Count")
-@Description({"The amount of online players. This can be changed in a",
+@Description({
+		"The amount of online players. This can be changed in a " +
 		"<a href='events.html#server_list_ping'>server list ping</a> event only to show fake online player amount.",
-		"'real online player count' always returns the real count of online players and can't be changed.",
-		"",
-		"Fake online player count requires PaperSpigot 1.12.2+."})
-@Examples({"on server list ping:",
-		"	# This will make the max players count 5 if there are 4 players online.",
-		"	set the fake max players count to (online players count + 1)"})
+		"<code>real online player count</code> always return the real count of online players and can't be changed."
+})
+@Examples({
+		"on server list ping:",
+			"\t# This will make the max players count 5 if there are 4 players online.",
+			"\tset the fake max players count to (online players count + 1)"
```

Comment:
`online players count` is not valid is it?

Reply:
true 😬